docs/ui/components/Navigation/Navigation.tsx

```diff
@@ -15,14 +19,27 @@ export type NavigationProps = {
 export function Navigation({ routes }: NavigationProps) {
   const router = useRouter();
   const activeRoutes = useMemo(() => findActiveRoute(routes, router.pathname), [router.pathname]);
+  const persistScroll = usePersistScroll('navigation');
+
   return (
-    <nav>
-      <ApiVersionSelect />
-      {routes.map(route => navigationRenderer(route, activeRoutes))}
+    <nav css={navigationStyle}>
+      <LayoutScroll {...persistScroll}>
+        <ApiVersionSelect />
+        {routes.map(route => navigationRenderer(route, activeRoutes))}
+      </LayoutScroll>
     </nav>
   );
 }
 
+const navigationStyle = css({
+  width: 280,
+  height: '100%',
+  backgroundColor: theme.background.secondary,
+  '[data-expo-theme="dark"] &': {
+    backgroundColor: theme.background.default,
+  },
```

Comment:
@jonsamp Not sure if this is what you had in mind, but this works nicely. We could also expose the data property in the styleguide, to keep it in sync

Reply:
ah this works! i haven't used this particular implementation and it's a lot more concise than the solution i was thinking of. Thanks!

What would you want exported from styleguide?

Reply:
Could be something like `${darkModeAttribute} &`, not sure if that over engineers it but this seems easily replaceable in the future too.